test/ts-benchmarks/bench.ts

```diff
@@ -0,0 +1,55 @@
+import { bench } from "@ark/attest";
+import { syncToRecs } from "@latticexyz/store-sync/recs";
+import { syncToZustand } from "@latticexyz/store-sync/zustand";
+import { Hex, type PublicClient } from "viem";
+import { defineWorld } from "@latticexyz/world";
+import { createWorld } from "@latticexyz/recs";
+
+bench("syncToRecs", async () => {
+  const config = defineWorld({
+    namespace: "demo",
+    tables: {
+      Position: {
+        schema: {
+          entity: "address",
+          x: "int32",
+          y: "int32",
+        },
+        key: ["entity"],
+      },
+    },
+  });
+
+  const t = await syncToRecs({
+    world: createWorld(),
+    config,
+    address: {} as Hex,
+    publicClient: {} as PublicClient,
+  });
+
+  return t;
+}).types([20704, "instantiations"]);
```

Comment:
I saw this 2x when using TS types from source as opposed to DTS. I wonder if we should have benchmarks for both cases?

Reply:
Yeah I saw this as well. The differences usually aren't so drastic for me but it makes sense in a repo like this where you have a lot of complex function returns etc, because it's checking your own .ts code as well.

The .d.ts test is going to be the more relevant for your users, but including paths would still give you something to iterate on so I think it could be reasonable to include both.